_review/qec-publiccloud.toml

```
@@ -47,4 +49,62 @@ Params = { flavor = "GCE-Updates", build = "%yesterday%-1" }
 [[Groups]]
 Name = "GCE-BYOS-Updates"
 Params = { flavor = "GCE-BYOS-Updates", build = "%yesterday%-1" }
+[[Groups]]
+Name = "GCE-CHOST-BYOS-Updates"
+Params = { flavor = "GCE-CHOST-BYOS-Updates", build = "%yesterday%-1" }
+
+[[Groups]]
+Name = "Server-DVD-Updates"
```

Server-DVD-Updates? Does that even make sense for Publiccloud?

---

Let's rename the name to something more descriptive:

```suggestion
Name = "Publiccloud CLI tools"
```

---

May be 😉

---

Done.